### DISCUSSION TITLE
Auto 模式创建的项目 ErrorLogger 问题

### DISCUSSION THREAD

**Discussion by yc-2503** [dotnetcore/BootstrapBlazor#4447] [Q&A]
### Which class is this unit test associated with?

在 InteraciveServer 项目的 MainLayout 定义的 OnErrorHandleAsync 是不是捕捉不到 InteractiveWebAssembly 项目的 异常啊？

**Comment by bb-auto** [comment]
@yc-2503 Thank you for submitting. We will give feedback later.

**Comment by ArgoZhang** [comment]
@yc-2503 感谢提交 Issue 这中问题类的，建议使用 Discussions 进行沟通
